Projects/UOContent/Mobiles/Monsters/ML/Twisted Weald/Changeling.cs

```diff
@@ -101,7 +97,7 @@ public Mobile MorphedInto
                         m_LastMorph = Core.Now;
                     }
 
-                    m_MorphedInto = value;
+                    _morphedInto = value;
```

Comment:
Add `this.MarkDirty();` to the end of this function.

Reply:
do you mean at the end of the set or inside the last if?

Reply:
Inside the if. Thanks for clarifying.